class-14.md

```diff
@@ -0,0 +1,85 @@
+# Intro to Blockchain Development
+
+### Class 14 - Wallets 
+
+We'll jump back to how cryptocurrency wallets are implemented, to give a better idea for users.
+
+Reading & Videos:
+
+- [Mastering Bitcoin Chapter 5](https://github.com/bitcoinbook/bitcoinbook/blob/develop/ch05.asciidoc)* - A deeper look at how a wallet works in bitcoin
+
+
+
+### Intro To Wallets
+
+We looked before briefly at how funds are stored in a wallet, different community proposals to enhance storage of funds, and how SPV worked to rapidly verify transactions. Now, we will dig a bit farther into the process, and explore how to interact with transactions at a programmatic level as well.
```

Comment:
`Now, we will dig a bit farther into the process, and explore how to interact with transactions at a programmatic level as well.`

`Now we can dig further to explore how to interact with transactions at a programmatic level.`